visidata/guides/DirSheet.md

```diff
@@ -0,0 +1,35 @@
+---
+sheettype: DirSheet
+---
+# Directory Sheet
+
+The **DirSheet** is a display of files and folders in the *source* directory.
+
+To load a **DirSheet**:
+
+- provide VisiData a directory as input (e.g. [:code]vd sample_data/[/])
+- {help.commands.open_dir_current} (equivalent to [:code]vd .[/])
+
+Pass the flag [:code]-r[/] ([:code]--recursive[/]) to display all of the files in all subfolders.
+
+Basic usage of the **DirSheet** is to find and load files within VisiData:
+
+- {help.commands.open_row_file}
+- {help.commands.open_rows}
+- ([:longname]open-dir-parent[/]) to open parent directory
```

Comment:
can we use `{help.commands.open_dir_parent}` here?

Reply:
Yes! I think this was before we added support for unbound.

Reply:
Oh no! It was because the keystroke ``` broke things in the past. It seems to render correctly now.